docs/guide/extending_online_editor.md

```diff
@@ -12,10 +13,15 @@ Refer to [Alloy Editor documentation](https://alloyeditor.com/docs/develop/) to
 ## Custom tags
 
 Custom tags enable you to add more features to the Rich Text editor beyond the built-in ones.
+They are configured under the `ezrichtext` key. 
 
-Custom tags are configured under the `ezrichtext` key (here on the example of a YouTube tag):
+If you want to learn how to apply them to your installation follow [Creating a custom tag tutorial](../../tutorials/extending_admin_ui/6_adding_a_custom_tag).
+ 
+### YouTube tag
```

Comment:
Maybe "Example: YouTube tag" and the same below with factbox? So it's clear it just an example you can adapt to your needs.